docs/overview_API.ipynb

```diff
@@ -1583,19 +1589,37 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Y2R"
+    "### Y2R\n",
+    "\n",
+    "Years-to-Recovery (Y2R) is a recovery metric the measures length of time (years) for a given pixel to reach 80% of it's recovery target value (Wulder et al., 2019; Frazier et al., 2018, De Keersmaecker et al., 2022). Given a recovery window that is $i$ years long, pixels that have recovered will have value between 0 - $i$, representing the number of years it tookfor the pixel to hit 80% of their recovery target. A value of 0 means that the pixel started at or above the threshold at the start of the recovery window, 1 means it took 1 year to reach the threshold, etc. Pixels that have not reached 80% of the recovery target at any point within the recovery window are given a value of -9999. "
```

Comment:
@melissabirch I've added you as reviewer to get your OK on the changes to the overview_API, particularly this paragraph describing Y2R and our implementation's outputs.